models/repo_watch.go

```diff
@@ -51,7 +51,11 @@ func WatchRepo(userID, repoID int64, watch bool) (err error) {
 
 func getWatchers(e Engine, repoID int64) ([]*Watch, error) {
 	watches := make([]*Watch, 0, 10)
-	return watches, e.Find(&watches, &Watch{RepoID: repoID})
+	return watches, e.Where("`watch`.repo_id=?", repoID).
+		And("`user`.is_active=?", true).
+		And("`user`.prohibit_login=?", false).
```

Comment:
I don't think user who is `prohibit_login` will be moved from watchers UI.

Reply:
The method is also used in https://github.com/go-gitea/gitea/blob/f61a1d210c4ed6f616b4bee25e8231b5bc67a640/models/notification.go#L104

Perhaps I should add a second method?

Reply:
Yes, add additional parameter to function to know in to return all as it was or just active ones

Reply:
@lunny @lafriks I figured out that the view isn't using this method anyways. The view is using the method from here https://github.com/daviian/gitea/blob/e3f679f39da7985193cec28a7c21fbc16ab2310d/models/repo_watch.go#L67

The method in question is only used by notification and mail

Reply:
@lunny Any response?

Reply:
@daviian OK.